### DISCUSSION TITLE
Google Cloud Stograge

### DISCUSSION THREAD

**Discussion by triadmoko** [pocketbase/pocketbase#1596] [Q&A]
can i integrate my storage using google cloud storage?

**Comment by ganigeorgiev** [comment]
I haven't personally tested it, but based on this article - https://cloud.google.com/storage/docs/aws-simple-migration and the Go code example, it should be possible to use the existing PocketBase S3 integration with GCS, but some configurations in your cloud dashboard may be required.

**Reply by rodydavis** [reply]
I was able to use GCP no problem with the article mentioned!

https://www.ibm.com/support/pages/using-google-cloud-storage-s3-advanced-storage-device-content-platform-engine
